mobile/src/libs/components/label/label.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Text, View } from '#libs/components/components';
+
+import { styles } from './styles';
+
+const Label: React.FC = () => {
```

Comment:
I don't think it should be a common component

Reply:
it is user for two screens for now. or should i still remove it to each component (sign in and sign up) ?

Reply:
It is only used in auth screens, maybe you can move it to #screens/auth/components